config/formats.js

```diff
@@ -280,6 +280,9 @@ let Formats = [
 		gameType: 'doubles',
 		ruleset: ['Standard Doubles'],
 		banlist: ['DUber', 'Beat Up'],
+		onBegin() {
+			if (this.rated && this.format.id === 'gen8doublesou') this.add('html', `<div class="broadcast-red"><strong>Doubles OU is currently suspecting Dynamax! For information on how to participate check out the <a href="https://www.smogon.com/forums/threads/3661422/">suspect thread</a>.</strong></div>`);
```

Comment:
```suggestion
			if (this.rated && this.format.id === 'gen8doublesou') {
				this.add('html', `<div class="broadcast-red"><strong>Doubles OU is currently suspecting Dynamax! For information on how to participate check out the <a href="https://www.smogon.com/forums/threads/3661422/">suspect thread</a>.</strong></div>`);
			}
```

Does the linter not check formats? This should be over line length limit...

Reply:
Apparently not because this line is pretty much copy/pasted for every suspect. There's a couple in formats currently in fact, so I'm just gonna merge the original for consistency.

Reply:
Just fix them all :[

Reply:
I already told you zarel, the linter doesnt check line length for JS files, so there's no need for this to be fixed

Reply:
The linter doesn't check line length _yet_, I want to turn it on eventually, and the more of this sort of thing we can fix ahead of time, the better.